multiqc/modules/mosaicatcher/mosaicatcher.py

```diff
@@ -0,0 +1,183 @@
+import logging
+from collections import OrderedDict
+
+from multiqc.modules.base_module import BaseMultiqcModule
+from multiqc.plots import bargraph, table
+
+log = logging.getLogger(__name__)
+
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        super(MultiqcModule, self).__init__(
+            name="MosaiCatcher",
+            anchor="mosaicatcher",
+            href="https://github.com/friendsofstrandseq/mosaicatcher",
+            info=(
+                "Mosaicatcher counts Strand-seq reads and classifies strand states "
+                "of each chromosome in each cell using a Hidden Markov Model."
+            ),
+            doi="10.1038/s41587-019-0366-x",
+        )
+        log_files = self.find_log_files("mosaicatcher", filehandles=False)
+
+        samples = dict()
+        for f in list(log_files):
+            if not f or "f" not in f:
+                log.warning("Malformed log file object: {}".format(f))
+            else:
+                print("Parsing log file:", f["fn"])
+                samples = self._parse_samples(f, samples)
+                self.add_data_source(f)
+
+        self._samples = samples
+
+        # Filter to strip out ignored sample names
+        self._samples = self.ignore_samples(self._samples)
+
+        if len(self._samples) == 0:
+            raise UserWarning
+
+        self._add_table(samples)
+        self._add_coverage_plot(samples)
+
+        self.write_data_file(self._samples, "multiqc_mosaicatcher")
+        log.info("Found {} samples".format(len(samples)))
+
+    def _add_table(self, samples):
+        self.add_section(
+            plot=table.plot(data=samples, headers=self._setup_headers()),
```

Comment:
You need to specify the ID for the table (the CI checks fail because of that).

You can also set the namespace globally for the table.

```suggestion
            plot=table.plot(
                data=samples, 
                headers=self._setup_headers(),
                pconfig={"id": "mosaicatcher_table", "namespace": "MosaiCatcher"}
            ),
```